.github/ISSUE_TEMPLATE/crx-tut-request.yaml

```diff
@@ -0,0 +1,61 @@
+name: Extension tutorial request
+description: Use this form to request a new Chrome Extension tutorial
+title: "[EXT-TUTORIAL] <title>"
+labels: [content, extensions]
+assignees: amysteam
+body:
+- type: markdown
+  attributes:
+    value: "## Thank you for sharing your feedback. Please fill out this form so that we can better understand what extension tutorial you are looking for:"
```

Comment:
Forgive my unfamiliarity with this type of file. What does '##' do?

Reply:
It's an H2 title, it looks like this:

<img width="988" alt="Screen Shot 2022-10-31 at 15 30 36" src="https://user-images.githubusercontent.com/37001393/199104749-e7df5103-3be0-4548-9303-55ce2caf3f5f.png">